src/common.js

```diff
@@ -14,29 +14,31 @@ export const Link = styled.a`
   margin: 0.5em;
   display: inline-block;
 
-  & svg {
```

Comment:
@SaraVieira hey! Did you happen to include this because you came to styled-components from using JSS? I just ran into an issue with JSS styling on a project where I was missing the `&` and it was funking up my styles:

![image](https://user-images.githubusercontent.com/5795227/39651172-2d4cf8c2-4feb-11e8-8d6d-1cb76b3779f8.png)

And when I realised what the solution was, I remembered seeing that in your lib 😛 

(Nothing at all to do with react-social-sharing, just a mere question! 👋 )

Reply:
Hey !

No this is a thing that I got the habit because of postcss-next for like a year.
http://cssnext.io/features/#nesting

They only allow nesting with the & before so I just got in the habit of putting it everywhere. 
It can't hurt ahah